mteb/tasks/STS/ron/RonSTS.py

```diff
@@ -0,0 +1,48 @@
+from __future__ import annotations
+
+from mteb.abstasks.TaskMetadata import TaskMetadata
+
+from ....abstasks.AbsTaskSTS import AbsTaskSTS
+
+
+class RonSTS(AbsTaskSTS):
+    metadata = TaskMetadata(
+        name="RonSTS",
+        dataset={
+            "path": "ro_sts",
+            "revision": "41a33183b739070f3d46d9d446492c1d2f98ce1a",
+        },
+        description="High-quality Romanian translation of STSBenchmark.",
+        reference="https://openreview.net/forum?id=JH61CD7afTv",
+        type="STS",
+        category="s2s",
+        eval_splits=["test"],
+        eval_langs=["ron-Latn"],
+        main_score="cosine_spearman",
+        date=None,
+        form=["written"],
+        domains=["News", "Social", "Web"],  # web for image captions
+        task_subtypes=None,
```

Comment:
Does this match anything listed in `mteb/abstasks/TaskMetadata.py`?

Reply:
nope. it's a semantic similarity task (no other STS task has values for this field)